vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -8792,6 +8792,24 @@ unit:GigaJ
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Gigajoule"@en ;
 .
+unit:GigaJ-PER-HR
+  a qudt:Unit ;
+  dcterms:description "SI derived unit Gigajoule divided by the 3600 times the SI base unit second"^^rdf:HTML ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 3600000000.0 ;
```

Comment:
Three more zeroes, right? 10**9 for the Giga and 3.6x10*3 for the HR to SEC conversion.